src/main/java/eventTask.java

```diff
@@ -0,0 +1,14 @@
+public class eventTask extends Task{
+    private String starts,ends;
```

Comment:
You may put a whitespace after the comma.
```suggestion
    private String starts, ends;
```